### PR TITLE
Feat/new markdown editor poc

### PR DESCRIPTION
**Pull request by Darthmaul**
Ok this new markdown editor should solve all of our problems with the editor thus far.

The editor is this guy: https://github.com/code-farmer-i/vue-markdown-editor

The implementation is a lot simpler and seems to work with most test cases so far. No debounce on the description saving right now, but can add if we feel it's necessary

### CODE REVIEW

**Comment by Darthmaul** [comment]
Changes requested:

- [ ] Improve the styling of the editor (background color)
- [ ] See if we can fix the tooltip fonts
- [ ] Remove image upload button
- [ ] Staging test with data from previous loadouts

**Comment by Darthmaul** [comment]
I think we should be fine with a quick stage test with some loadouts from Prod to check descriptions, but overall I feel like it's ready.

@chaseconey if you have time for a review this week I think it's ready

**Comment by Darthmaul** [comment]
Action: Do the previewer before submitting PR - then do that again

**Comment by Darthmaul** [comment]
Previewer is deploying to stage, doing testing now

File: resources/js/components/LoadoutPreviewPage.vue [hunk 1]
```diff
@@ -46,16 +46,17 @@
             <h2 @click="guideIsOpen = !guideIsOpen" class="text-karl-orange text-xl uppercase p-4">Loadout Guide <i
                 :class="guideIsOpen ? 'fas fa-chevron-down invertIcon': 'fas fa-chevron-down'"></i></h2>
             <div v-show="guideIsOpen" class="p-6">
-                <viewer :initialValue="loadoutData.description"/>
+                <!-- <viewer :initialValue="loadoutData.description"/> -->
```
Comment (chaseconey) [inline resources/js/components/LoadoutPreviewPage.vue:49]: Can remove this lil fella

File: resources/js/components/LoadoutPreviewPage.vue [hunk 2]
```diff
@@ -46,16 +46,17 @@
...
         </div>
 
     </div>
 </template>
 
 <script>
-import '@toast-ui/editor/dist/toastui-editor-viewer.css';
-import {Viewer} from '@toast-ui/vue-editor';
+// import '@toast-ui/editor/dist/toastui-editor-viewer.css';
+// import {Viewer} from '@toast-ui/vue-editor';
```
Comment (chaseconey) [inline resources/js/components/LoadoutPreviewPage.vue:59]: and these

File: resources/js/app.js [hunk 3]
```diff
@@ -55,6 +55,32 @@ import VueApollo from "vue-apollo";
...
+import 'prismjs/components/prism-json';
+
+VueMarkdownEditor.use(vuepressTheme, {
+  Prism,
+});
+VueMarkdownEditor.lang.use('en-US', enUS);
+
+Vue.use(VueMarkdownEditor);
+VMdPreview.use(githubTheme);
+Vue.use(VMdPreview);
```
Comment (chaseconey) [inline resources/js/app.js:83]: We could probably extract this into a component, but we can do that later I suppose unless you are down to give it a shot

**Review by chaseconey — CHANGES_REQUESTED** [review]

File: resources/js/app.js (hunk 3 above)
Comment (Darthmaul) [reply resources/js/app.js:83]: I've never done that before in the context of the `app.js` file. I would prefer to punt on that one until I can get a better understanding

**Review by chaseconey — APPROVED** [review]

**Pull request merged**